src/components/JourneyBlock.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+export interface JourneyEntry {
+    arrivalTime: string;
+    departureTime: string;
+    destinationStation: {'crs': string; 'displayName': string};
+    isFastestJourney: boolean;
+    journeyDurationInMinutes: number;
+    originStation: {'crs': string; 'displayName': string};
+    stationMessages: any[];
+    tickets: any[];
+}
+
+interface JourneyBlockProps {
+    journeyData: JourneyEntry;
```

Comment:
Do we need to pass around this JourneyBlockProps wrapper if we only care about JourneyEntry?

Reply:
I believe the values always need to be unpacked regardless of how they're passed in? And so this would be the alternative format:

![image](https://github.com/LeoPhh/bootcamp-react-trainboard/assets/135320134/64d3b871-2d55-48e0-bde2-d866b949ca6a)

We think this new format is slightly worse. If there is a cleaner way to do it we'd love to learn it :)